MekHQ/src/mekhq/gui/dialog/NewAtBContractDialog.java

```diff
@@ -577,8 +582,8 @@ protected void doUpdateContract(Object source) {
     		contract.setEnemyQuality(cbEnemyQuality.getSelectedIndex());
     	}
     	
-   		updatePaymentMultiplier();
-   		super.doUpdateContract(source);
+    	if (needUpdatePayment) { updatePaymentMultiplier(); }
```

Comment:
This if statement also should be changed to reflect our coding style standards:
    if (conditional) {
        code_here
    }

Rather than the one line version it currently has.

Reply:
Will do and update.